### ISSUE TITLE
Skin switching 

### ISSUE THREAD

**Issue by stgraveyard** [atari-legend/atari-legend#180]
Hi Nico, 

I know you are not really into this stuff, but I'm gonna ask anyway. When AL went 'live' in october 2017, we had 3 skins. But I only took the Tron like skin live. There was another skin in GEM color. I was wondering now, since you completely reprogrammed it all and since the CPANEL is getting its own thing, would it be hard to add the skin switching feature to the main site. I mean, all graphics and colors are in the CSS. Maybe we could add the GEM look and integrate it into your new CSS and code? 

It was just an idea. We could have people with an account be able to save their favorite skin for each time they visit. If you think it is stupid and don't feel up to it, not to worry, I understand. It just popped in my head, also because LGD has been dead for years now ...

Cheers,
Maarten

**Comment by nguillaumin** [comment]
Yeah, I'm aware of this I remember the other skins. Skin switching in itself would not be difficult, the real challenge is to rebuild the GEM skin on the new site since the structure has completely changed. The structure is quite simpler than before so re-doing the skin will be less work, but still it is some work and CSS / graphics is not my strong suit.

So unless someone can do the CSS work to build the skin we're probably stuck on that one...

**Comment by stgraveyard** [comment]
OK Nico, I currently don't have time for this (I'm already working on a new video! ;) ) BUT ... would it perhaps be possible for me to use the dev system for this. I mean, if I take a copy of the existing CSS and start comparing this with the old GEM css, than alter the copy of the css, upload it to dev, test the skin, alter css, upload, test, alter, upload, test ... This way I don't need to install any tools and I can slowly build the new skin? I mean, restoring the currect tron css to dev is 2 seconds of work so I guess it would be harmfull if I could work like this maybe somewhere in the future?

**Comment by nguillaumin** [comment]
I guess it would be possible, but I use the dev site sometimes to test new implementations so it would conflict with that.

If you're really interested I think it would be better if I create a separate site, like I did when I rewrote it, like prototype.atarilegend.com or similar...

**Comment by stgraveyard** [comment]
Good idea. If you set it up, I will be working slowly on it as time passes by, but there is now hurry. Could be cool!

**Comment by nguillaumin** [comment]
I thought about this a bit more but in fact it won't work because the CSS is build with SASS, so it needs to be compiled to generate the final CSS. Plus it extends the Bootstrap CSS (from the SASS sources too) so you cannot really work in "isolation" unfortunately... Best would be to use the Docker based development environment and start from the original plain Bootstrap CSS, then add whatever is needed to customize it (which is exactly what I did to reach the current skin).

**Comment by stgraveyard** [comment]
Ok, I understand. You can close this one. I will not have time to do this in that way as I will focus on making videos. I don't want to install the whole thing on my laptop...
